test/cmdlineTests/name_simplifier/output

```diff
@@ -80,10 +80,10 @@ object "C_59" {
             {
                 memPtr := mload(64)
                 let newFreePtr := add(memPtr, and(add(size, 31), not(31)))
-                if or(gt(newFreePtr, 0xffffffffffffffff), lt(newFreePtr, memPtr)) { panic_error_0x41() }
+                if or(gt(newFreePtr, 0xffffffffffffffff), lt(newFreePtr, memPtr)) { panic_errorx41() }
                 mstore(64, newFreePtr)
             }
-            function copy_literal_to_memory_64902fd228f7ef267f3b474dd6ef84bae434cf5546eee948e7ca26df3eda1927() -> memPtr
+            function copy_literal_to_memoryfd228f7ef267f3b474dd6ef84bae434cf5546eee948e7ca26df3eda1927() -> memPtr
```

Comment:
Old more readable because the last part of the new name mixes name with (what is presumably) the literal to be copied.